proxy/http/HttpConfig.h

```diff
@@ -618,19 +619,19 @@ struct OverridableHttpConfigParams {
   /////////////////////////////////////////////////
   MgmtByte response_suppression_mode = 0; // proxy.config.body_factory.response_suppression_mode
 
+  //////////////////
+  // Redirection  //
+  //////////////////
+  MgmtByte redirect_use_orig_cache_key = 0;
+  MgmtInt number_of_redirections       = 0;
+
```

Comment:
It would be zwoopy of me to say you need a separate PR to move these line.

Reply:
@ywkaras Well played. I can move these out if you prefer, having these out of "byte order" so to speak just rubs me the wrong way since they add so much uneccesary padding to the structs.